<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UFormSalaryEditTabelWidget</class>
 <widget class="QWidget" name="UFormSalaryEditTabelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Повний Табель</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../uPictRc.qrc">
    <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/7days-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/7days-256.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_filter">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Фільтр: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tabelDate">
         <property name="text">
          <string>Дата табеля</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_tabelDate">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="displayFormat">
          <string>MM.yyyy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pidrozdil">
         <property name="text">
          <string>Підрозділ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_pidrozdil">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reloadTabel">
         <property name="text">
          <string>Перечитати табель</string>
         </property>
         <property name="icon">
          <iconset resource="../uPictRc.qrc">
           <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_reload-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_reload-256.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_printTabel">
         <property name="text">
          <string>Друк табеля</string>
         </property>
         <property name="icon">
          <iconset resource="../uPictRc.qrc">
           <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/fileprint-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/fileprint-256.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_poshuk">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пошук: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dateFilter">
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_dateFilter">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="displayFormat">
        <string>dd.MM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_vodiyFilter">
       <property name="text">
        <string>Працівник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_nprFilter">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_findCell">
       <property name="text">
        <string>Знайти</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/search-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/search-256.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_tabel">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_reloadTabel">
   <property name="icon">
    <iconset resource="../uPictRc.qrc">
     <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/reload-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/reload-256.png</iconset>
   </property>
   <property name="text">
    <string>Поновити показники табеля</string>
   </property>
   <property name="toolTip">
    <string>Поновлення показників табеля з бази даних без їх зміни</string>
   </property>
  </action>
  <action name="action_printTabel">
   <property name="icon">
    <iconset resource="../uPictRc.qrc">
     <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/fileprint-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/fileprint-256.png</iconset>
   </property>
   <property name="text">
    <string>Друк табеля</string>
   </property>
   <property name="toolTip">
    <string>Друк табеля за поточний місяць</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>dateEdit_tabelDate</tabstop>
  <tabstop>comboBox_pidrozdil</tabstop>
  <tabstop>pushButton_reloadTabel</tabstop>
  <tabstop>tableWidget_tabel</tabstop>
  <tabstop>dateEdit_dateFilter</tabstop>
  <tabstop>comboBox_nprFilter</tabstop>
  <tabstop>pushButton_findCell</tabstop>
 </tabstops>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
